examples/ecs-target/main.tf

```diff
@@ -0,0 +1,325 @@
+provider "aws" {
+  region = "ap-southeast-1"
+
+  # Make it faster by skipping something
+  skip_get_ec2_platforms      = true
+  skip_metadata_api_check     = true
+  skip_region_validation      = true
+  skip_credentials_validation = true
+  skip_requesting_account_id  = true
+}
+
+module "eventbridge" {
+  source = "../../"
+
+  bus_name = "${random_pet.this.id}-bus"
+
+  attach_ecs_policy = true
+  ecs_target_arns   = [aws_ecs_task_definition.hello_world.arn]
+
+  attach_cloudwatch_policy = true
+  cloudwatch_target_arns   = [aws_cloudwatch_log_group.this.arn]
+
+  rules = {
+    logs = {
+      description = "Capture S3 Events"
+      event_pattern = jsonencode({
+        "source" : ["aws.s3"],
+        "detail-type" : ["AWS API Call via CloudTrail"],
+        "detail" : {
+          "eventSource" : ["s3.amazonaws.com"],
+          "eventName" : ["PutObject"],
+          "requestParameters" : {
+            "bucketName" : [module.source_bucket.s3_bucket_id]
+          }
+        }
+      })
+    }
+  }
+
+  targets = {
```

Comment:
I like the examples but this one is just huge.

Let's merge just the required parts related to ECS target with the complete example:
1. We can reuse existing rule from `rules` in complete example.
2. S3 <> Cloudtrail integration does not really belong to the example of EventBridge, so we need to eliminate it in this case (it may be more relevant to S3 bucket examples in S3 bucket module but not here).

Reply:
Fair point, let me take care of that.